### DISCUSSION TITLE
Increase docker container size in Ubuntu

### DISCUSSION THREAD

**Discussion by jimcobber** [utmstack/UTMStack#700] [General]
Hi, all. Does anyone know what the process is to increase the UTMstack container size? Currently it is at the default100mb. and I'm running it on Ubuntu 22.04. Thanks!

**Comment by c3s4rfred** [comment]
Hi, @jimcobber, please follow our installation guides -> https://docs.utmstack.com/, there you can find information about UTMStack system requirements and guidelines for a correct installation, normally if your question is not in the site means that:  or you don't need to do it or is out of UTMStack's range, consider to check docker official documentation. Best regards.

**Reply by jimcobber** [reply]
I did the UTMstack install directions and it's working fine, it just is running out of space even before the 30 days of data limitations of the free version. I installed it on a 1TB volume, but the container size is 100mb. Can you please direct me to the page in the official documentation that explains how to increase space for log files?
